editor/styling/stylesheets/_editor.scss

```diff
@@ -145,19 +145,7 @@ Links:
     -fx-background-color: -cross-color;
     -fx-max-width: 10px;
     -fx-max-height: 10px;
-    -fx-shape: "M 2 0
-                L  5  3
-                L  8  0
-                L 10  2
-                L  7  5
-                L 10  8
-                L  8 10
-                L  5  7
-                L  2 10
-                L  0  8
-                L  3  5
-                L  0  2
-                Z";
+    -fx-shape: "M 2 0 L  5  3 L  8  0 L 10  2 L  7  5 L 10  8 L  8 10 L  5  7 L  2 10 L  0  8 L  3  5 L  0  2 Z";
```

Comment:
It was nicer before, but Dart Sass doesn't like it :/

Reply:
Maybe we should remove the formatting whitespace (i.e. the double-spaces) altogether?